keyboards/era/linx3/n8x/info.json

```
@@ -0,0 +1,135 @@
+{
+    "manufacturer": "eerraa",
+    "keyboard_name": "N8X",
+    "maintainer": "eerraa",
+    "backlight": {
+        "breathing": true,
+        "breathing_period": 5,
+        "levels": 10,
+        "pin": "GP3"
+    },
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": true,
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "GP11",
+        "scroll_lock": "GP8"
+    },
+    "matrix_pins": {
+        "cols": ["GP29", "GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP22", "GP20", "GP19", "GP18", "GP17", "GP16", "GP21", "GP1", "GP0", "GP2"],
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP10", "GP9"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0007",
+        "vid": "0x4552"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [0, 2], "x": 2.25, "y": 0},
+                {"matrix": [0, 3], "x": 3.25, "y": 0},
+                {"matrix": [0, 4], "x": 4.25, "y": 0},
+                {"matrix": [0, 5], "x": 5.5, "y": 0},
+                {"matrix": [0, 6], "x": 6.5, "y": 0},
+                {"matrix": [0, 7], "x": 7.5, "y": 0},
+                {"matrix": [0, 8], "x": 8.5, "y": 0},
+                {"matrix": [0, 9], "x": 9.75, "y": 0},
+                {"matrix": [0, 10], "x": 10.75, "y": 0},
+                {"matrix": [0, 11], "x": 11.75, "y": 0},
+                {"matrix": [0, 12], "x": 12.75, "y": 0},
```

Without an image of PCB and based on written confirmation of a F12 top row, this suggestion is somewhat conjecture.
```suggestion
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6.5, "y": 0},
                {"matrix": [0, 7], "x": 7.5, "y": 0},
                {"matrix": [0, 8], "x": 8.5, "y": 0},
                {"matrix": [0, 9], "x": 9.5, "y": 0},
                {"matrix": [0, 10], "x": 11, "y": 0},
                {"matrix": [0, 11], "x": 12, "y": 0},
                {"matrix": [0, 12], "x": 13, "y": 0},
```